internal/remote/graphql/usage.go

```diff
@@ -2,9 +2,10 @@ package graphql
 
 // Usage describes organization usage statistics.
 type Usage struct {
-	OrganizationID     string `graphql:"-"`
-	DeploymentCount    *int   `json:"deploymentCount"`
-	MemberCount        *int   `json:"memberCount"`
-	NodeCount          *int   `json:"nodeCount"`
-	CloudSlackUseCount *int   `json:"cloudSlackUseCount"`
+	DeploymentCount    *int `json:"deploymentCount"`
+	MemberCount        *int `json:"memberCount"`
+	NodeCount          *int `json:"nodeCount"`
+	CloudSlackUseCount *int `json:"cloudSlackUseCount"`
+
+	// All internal, ignored fields are removed
```

Comment:
Do we need this comment? It seems autogenerated

Reply:
Oh, I've added it just to ensure we won't bloat models in Botkube Agent with some internal fields from Botkube Cloud 🙂